target/linux/ramips/dts/mt7628an_hilink_hlk-7628n.dts

```diff
@@ -29,13 +29,6 @@
 	};
 };
 
-&state_default {
-	gpio {
-		groups = "i2c";
-		function = "gpio";
-	};
-};
-
```

Comment:
Why is this removed?

Reply:
On HILINK-7628N, this is useless code.

Reply:
"Why is it green?" - "Because somebody painted it that way ..." :-(

Please tell me why you think this is useless... No, don't tell me. Put it in the commit message of your second commit!

Reply:
Looks like this shouldn't be removed but replaced by the correct values...